### PR TITLE
Zach CodeCov

### PR DESCRIPTION
**Pull request by hanzemeng**
Updated link to the right CodeCov page and changed the title of the readme file to current quarter's team.

### CODE REVIEW

**Review by notkevin12 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# team04-w22-5pm-courses
+# team01-s22-6pm-courses
```
Comment (andrewhlu) [inline README.md:1]: This is actually `proj01` instead of `team01`, but it's not urgent so you're welcome this later.

**Review by andrewhlu — DISMISSED** [review]
LGTM! One small correction below

**Review by andrewhlu — APPROVED** [review]
Looks good, thanks!

**Review by notkevin12 — APPROVED** [review]
LGTM

**Pull request merged**